src/app/auth-service.ts

```diff
@@ -14,18 +14,18 @@ export class AuthService {
     const auth = getAuth();
     createUserWithEmailAndPassword(auth, email, password)
     .then( res => {
-      console.log(res)
+      console.log(res);
     })
     .catch(error => {
-      console.log(error);
+      alert(error);
     })
   }
 
   loginUser(email: string, password: string) {
     const auth = getAuth();
     signInWithEmailAndPassword(auth, email, password)
     .then( res => {
-      console.log("login response: ",res.user)
+      localStorage.setItem("userId", res.user.email ? res.user.email: '');
```

Comment:
localStorage add on service

Reply:
Created service file and added localStorage methods.